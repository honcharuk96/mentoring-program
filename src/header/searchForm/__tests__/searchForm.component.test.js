import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SearchForm from '../searchForm.component';

const shallow = new ShallowRenderer();
const setPosterDataFromForm = jest.fn();

describe('<SearchForm />', () => {
  it('should render SearchForm withoutPosterData', () => {
    const header = shallow.render(<SearchForm setPosterDataFromForm={setPosterDataFromForm} />);
    expect(header).toMatchSnapshot();
  });
});
