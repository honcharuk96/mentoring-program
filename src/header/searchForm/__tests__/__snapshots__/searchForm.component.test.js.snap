// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchForm /> should render SearchForm withoutPosterData 1`] = `
<React.Fragment>
  <searchFormstyled__Search>
    <searchFormstyled__Text>
      FIND YOUR MOViE
    </searchFormstyled__Text>
    <searchFormstyled__Inputs
      onSubmit={[Function]}
    >
      <searchFormstyled__SearchInput
        onChange={[Function]}
        value=""
      />
      <RedButtonComponent
        click={[Function]}
        revertColor={false}
        text="search"
      />
    </searchFormstyled__Inputs>
  </searchFormstyled__Search>
</React.Fragment>
`;
